main.go

```diff
@@ -0,0 +1,182 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/skuid/changelog/src/changelog"
+	"github.com/skuid/changelog/src/linkStyle"
+	"github.com/skuid/changelog/src/writer"
+	flag "github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+var providers = []string{
+	"local",
+	"github",
+	// "bitbucket",
+	// "stash",
+	// "gitlab"
+}
+
+var (
+	subtitle          = flag.String("subtitle", "", "The release subtitle")
+	changelogFile     = flag.String("changelog", "", "The file to write. Defaults to STDOUT if not set.")
+	fromLatestTag     = flag.Bool("from-latest-tag", false, "If you use tags, set to true to get changes from latest tag.")
+	fromCommit        = flag.StringP("from", "f", "", "The beginning commit. Defaults to beginning of the repository history")
+	toCommit          = flag.StringP("to", "t", "HEAD", "The last commit.")
+	sinceTime         = flag.String("since", "", "Show commits more recent than a specific date. Use RFC3339 time '2017-08-01T00:00:00Z'. Takes precedence over to/from.")
+	untilTime         = flag.String("until", "", "Show commits older than a specific date. Defaults to current time if not set, but --since is. Takes precedence over to/from.")
+	version           = flag.StringP("version", "v", "", "The version you are creating")
+	repoLink          = flag.StringP("repo", "r", "", "The repository URL. Defaults to `$(git remote get-url origin)` if using a local provider")
+	includeAllCommits = flag.Bool("include-all", false, "Set to true to include all commits in the changelog. Commit messages that cannot be parsed will be placed in a section titled \"Unknown\".")
+
+	provider = flag.StringP("provider", "p", "local", fmt.Sprintf(`The provider to use. Must be one of %s`, strings.Join(providers, ", ")))
+
+	token = flag.String("token", "", "API token for remote provider. Only applies to github provider")
```

Comment:
I'm sure Viper handles this, but is there a corresponding environment variable for this?

Reply:
Yea, it'd just be `CHANGELOG_TOKEN`. I'll add that to the README